src/views/about/about.jsx

```diff
@@ -53,31 +53,62 @@ var About = React.createClass({
                                 webkitAllowFullScreen
                                 mozallowfullscreen
                                 allowFullScreen />
-                            <p><FormattedHTMLMessage id='about.literacyDescription' /></p>
+                            <p><FormattedMessage id='about.literacyDescription' /></p>
                         </li>
 
                         <li>
                             <h3><FormattedMessage id='about.aroundTheWorld' /></h3>
                             <img src="/images/about/around-the-world.png" alt="" />
-                            <p><FormattedHTMLMessage id='about.aroundTheWorldDescription' /></p>
+                            <p><FormattedMessage id='about.aroundTheWorldDescription' /></p>
```

Comment:
Values is the second argument to `FormattedMessage`, so it needs to be inside the component tag. Getting the indentation/formatting will probably help with finding all of these; Lines 62-69 should be formatted as:
```jsx
<p>
    <FormattedMessage 
        id='about.aroundTheWorldDescription'
        values={{
            translationLink: (
                <a href='http://wiki.scratch.mit.edu/wiki/How_to_Translate_Scratch\'>
                    <FormattedMessage id='about.translationLinkText' />
                </a>
            )
        }}
    />
</p>

Reply:
So I've just to make sure that everything is inside the ``<p>`` tag?